src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -55,6 +57,63 @@ public async Task<ActionResult> Index(Guid organisationId, ManageEvidenceNotesDi
             }
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Index(Guid organisationId)
+        {
+            return RedirectToAction("TransferEvidenceNote", "ManageEvidenceNotes", new { area = "Scheme", organisationId });
+        }
+
+        [HttpGet]
+        public async Task<ActionResult> TransferEvidenceNote(Guid organisationId)
+        {
+            using (var client = this.apiClient())
+            {
+                await SetBreadcrumb(organisationId, BreadCrumbConstant.SchemeManageEvidence);
+
+                var model = new TransferEvidenceNoteDataViewModel();
+                model.SchemasToDisplay = await GetApprovedSchemes();
+                return this.View(model);
+            }
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> TransferEvidenceNote(TransferEvidenceNoteDataViewModel model)
+        {
+            using (var client = apiClient())
+            {
+                var selectedCaterogyIds = model.CategoryValues.Where(c => c.Selected).Select(c => c.CategoryId).ToList();
+
+                if (ModelState.IsValid)
+                {
+                    var transferSelectedData = new TransferSelectedDataModel(selectedCaterogyIds, model.SelectedSchema.Value);
```

Comment:
We could probably just create an instance of TransferEvidenceNoteRequest and stick that in the session. Move this code creation into a Request Creator.

Reply:
As you have below :) I'd probably get rid of TransferSelectedDataModel as not sure its needed. You could just create your request from a TransferEvidenceNoteDataViewModel.